src/store/adjustVault.ts

```diff
@@ -41,6 +41,14 @@ export const vaultToAdjustAtom = atom(get => {
     manager.compoundedInterest,
   );
 
+  // Prevent divide-by-zero if no debt.
+  const collateralizationRatio = AmountMath.isEmpty(totalDebt)
+    ? makeRatioFromAmounts(
+        totalLockedValue,
+        AmountMath.make(totalDebt.brand, 1n),
+      )
+    : makeRatioFromAmounts(totalLockedValue, totalDebt);
```

Comment:
```suggestion
  const nonzeroDebt = AmountMath.max(totalDebt, AmountMath.make(totalDebt.brand, 1n));
```
Then the line below could stay with a small change,
```js
collateralizationRatio: makeRatioFromAmounts(totalLockedValue, nonzeroDebt),
```
But… why have a collaterialization ratio at all if there's no debt? I would expect `undefined` or `null` since it doesn't make any sense.

Reply:
Done.

Hmm yea this is just a safe way of handling it, what would we show instead? ∞?

Reply:
Maybe safe from runtime failure but these seems less safe about what we're communicating to the user.

∞ would be better than what's here. But I think if there's no debt then the collateralization ratio is not applicable, so "n/a". I'm happy to leave it to Product to figure out what to display but I do request that we don't make up denominators in order to avoid zero division.

Reply:
Yea just changed to make these undefined in that case and show "N/A". Seems like ts was having trouble inferring the types, thinking they were 'any', so I made them explicit so components would know they could be undefined.

Reply:
Actually seems like it was having trouble inferring the return type for `Ratio`, just changed it to be more explicit.